3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/GoogleAnalyticsAdapter.scala

```diff
@@ -494,7 +512,7 @@ object GoogleAnalyticsAdapter extends Adapter {
    * @return a translated params
    */
   private def translatePayload(
-    originalParams: Map[String, String],
+    originalParams: Map[String,   String],
```

Comment:
can we exclude this type of alignment? doesn't make a lot of sense to me, if we can't that's fine...